### ISSUE TITLE
Add a11y labels and a11y ids to accessory view items

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#18633]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-8406)

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

For QA: Please test that the next, previous, and done button as well as credit card plus address autofill buttons above the keyboard are accessible using voice over.

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Hello, Winnie Teichmannplease note that I tried to swipe to reach the buttons you mentioned but I believe there is still an issue with it.

Use saved password is read, The Up and Down button arrows are read but they are not displayed as being selected, the Done button is skipped and then the use can reach the keyboard elements by swiping.

Checked on v127 (41813) with iPhone 13 Pro (15.7.1).

[^RPReplay_Final1715940826.MP4]
